slick/src/main/scala/slick/jdbc/JdbcStatementBuilderComponent.scala

```diff
@@ -592,7 +592,7 @@ trait JdbcStatementBuilderComponent { self: JdbcProfile =>
   /** Builder for various DDL statements. */
   class TableDDLBuilder(val table: Table[_]) { self =>
     protected val tableNode = table.toNode.asInstanceOf[TableExpansion].table.asInstanceOf[TableNode]
-    protected val columns: Iterable[ColumnDDLBuilder] = table.create_*.map(fs => createColumnDDLBuilder(fs, table))
+    protected def columns: Iterable[ColumnDDLBuilder] = table.create_*.map(fs => createColumnDDLBuilder(fs, table))
```

Comment:
Can we have a comment here about why this is a `def` instead of a `val`?

Reply:
✅